### DISCUSSION TITLE
Main repo?

### DISCUSSION THREAD

**Discussion by hejtmii** [ibash/react-hotkeys#7] [General]
Hi, I see this repo contains the latest changes published on NPM, is this the main repo now? Could it be updated?

https://www.npmjs.com/package/react-hotkeys-ce

I submitted an issue to the original repo but I am not sure how to proceed when that repo seems inactive for a long time. Please advise

https://github.com/greena13/react-hotkeys/issues/339

**Comment by hejtmii** [comment]
I created the PR which fxes the issue https://github.com/ibash/react-hotkeys/pull/8

**Comment by ibash** [comment]
This isn't an official main repo, mostly just forked and added my own fixes. Just merged #8, will publish an update now, thank you!!

**Comment by ibash** [comment]
Updated!

**Comment by hejtmii** [comment]
Awesome, thank you!
